
From: Pavel Machek <pavel@ucw.cz>

I shot myself in the foot with swsusp, so I guess documenting that
particular trap is right thing to do (tm).  Somehow two copies of "radeon
hint" crept in; fix that, too.

Signed-off-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Documentation/power/swsusp.txt |    3 +++
 25-akpm/Documentation/power/video.txt  |    4 ----
 2 files changed, 3 insertions(+), 4 deletions(-)

diff -puN Documentation/power/swsusp.txt~swsusp-minor-docs-updates Documentation/power/swsusp.txt
--- 25/Documentation/power/swsusp.txt~swsusp-minor-docs-updates	Wed Jun 23 14:12:21 2004
+++ 25-akpm/Documentation/power/swsusp.txt	Wed Jun 23 14:12:21 2004
@@ -12,6 +12,9 @@ From kernel/suspend.c:
  *				...you'd better find out how to get along
  *				   without your data.
  *
+ * If you change kernel command line between suspend and resume...
+ *			        ...prepare for nasty fsck or worse.
+ *
  * (*) pm interface support is needed to make it safe.
 
 You need to append resume=/dev/your_swap_partition to kernel command
diff -puN Documentation/power/video.txt~swsusp-minor-docs-updates Documentation/power/video.txt
--- 25/Documentation/power/video.txt~swsusp-minor-docs-updates	Wed Jun 23 14:12:21 2004
+++ 25-akpm/Documentation/power/video.txt	Wed Jun 23 14:12:21 2004
@@ -30,10 +30,6 @@ There are three types of systems where v
   patched X, and plain text console (no vesafb or radeonfb), see
   http://www.doesi.gmxhome.de/linux/tm800s3/s3.html. (Acer TM 800)
 
-* radeon systems, where X can soft-boot your video card. You'll need
-  patched X, and plain text console (no vesafb or radeonfb), see
-  http://www.doesi.gmxhome.de/linux/tm800s3/s3.html. (Acer TM 800)
-
 Now, if you pass acpi_sleep=something, and it does not work with your
 bios, you'll get hard crash during resume. Be carefull.
 
_
